.travis.yml

```diff
@@ -10,6 +10,9 @@ matrix:
       env: PERCY_ENABLE=0
 script:
   - npm test
-  - cd packages/react-percy-storybook && yarn link
-  - cd ../../integration-tests && yarn && yarn link @percy-io/react-percy-storybook
-  - npm run storybook:percy
+  - >
```

Comment:
@fotinakis Thanks, have put this into scripts/run-integration-test-snapshots.sh.